### ISSUE TITLE
the problem of training set and testing set 

### ISSUE THREAD

**Issue by ohdoughnut** [girishp92/Human-activity-recognition-using-Recurrent-Neural-Nets-RNN-LSTM-and-Tensorflow-on-Smartphones#1]
Hello, in the process of iterative training, I see that the testing set is also being iterated. Do you use the testing set as validation set, or do the iterative testing set have other meanings? Thank you.
The relevant code is as follows：
`for i in range(1, N_EPOCHS + 1):
    for start, end in zip(range(0, train_count, BATCH_SIZE),
                          range(BATCH_SIZE, train_count + 1, BATCH_SIZE)):
        sess.run(optimizer, feed_dict={X:X_train[start:end],
                                       Y:y_train[start:end]})

    _, acc_train, loss_train = sess.run([pred_softmax, accuracy, loss], feed_dict={
        X: X_train, Y:y_train})
    _, acc_test, loss_test = sess.run([pred_softmax, accuracy, loss], feed_dict={
        X: X_test, Y:y_test})`
